ptx/sec_deriv_basic_rules.ptx

```diff
@@ -232,7 +232,17 @@
           <caption>A graph of <m>f(x) = x^3</m>, along with its derivative <m>\fp(x) = 3x^2</m> and its tangent line at <m>x=-1</m></caption>
           <!-- START figures/fig_xcubedwithderiv.tex -->
           <image xml:id="img_xcubedwithderiv" width="47%">
-            <description></description>
+            <shortdescription>
+              Graph of function x^3, its derivative and a tangent line drawn at point (-1,-1).
+            </shortdescription>
+            <description>
+              <p>
+                The y-axis is drawn between -4 and 4 and the x-axis is drawn between -2 and 2.
+                The graph of function <m>x^3</m> is a curve that increases to infinity from 0 in the first quadrant, and decreases from 0 to infinity in the third quadrant.
+                The derivative of the graph meets the curve at the origin and increases to infinity but is drawn above the gaph of the function.
+                A tangent line is drawn on the graph of the function at point <m>(-1,-1)</m>.
```

Comment:
I think it is relevant to also mention that the tangent line has positive slope.